scripts/link-validator.conf

```diff
@@ -39,7 +39,7 @@ site-link-validator {
   ]
 
   ignore-missing-local-files-regex = ""
-  // e.g. "^api/alpakka/snapshot/pekko/stream/alpakka/googlecloud/storage/impl/Formats.*"
+  // e.g. "^api/connectors/snapshot/pekko/stream/connectors/googlecloud/storage/impl/Formats.*"
```

Comment:
Is that really correct? But maybe doesn't matter right now...

Reply:
I'll leave this open while I look at something else but I'll probably revert it as you suggest

Reply:
I reverted this line